cli/CampaignConfigValidation/ValidateCampaignConfigCommand.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Cli\CampaignConfigValidation;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+use WMDE\Fundraising\Frontend\Factories\FunFunFactory;
+use WMDE\Fundraising\Frontend\Infrastructure\ConfigReader;
+
+/**
+ * @license GNU GPL v2+
+ */
+class ValidateCampaignConfigCommand extends Command {
+
+	const NAME = 'app:validate:campaigns';
+	const DEFAULT_SCHEMA = __DIR__ . '/../../app/config/schema.json';
+	const ERROR_RETURN_CODE = 1;
+	const OK_RETURN_CODE = 0;
+
+	protected function configure(): void {
+		$this->setName( self::NAME )
+			->setDescription( 'Validate campaign configuration files' )
+			->setHelp(
+				'This command validates the Campaign configuration files for errors after they have been merged'
+			)
+			->setDefinition(
+				new InputDefinition(
+					[
+						new InputArgument(
+							'environment',
+							InputArgument::OPTIONAL,
+							'Specify a specific campaign environment which is to be tested',
+							'prod'
+						)
+					]
+				)
+			);
+	}
+
+	protected function execute( InputInterface $input, OutputInterface $output ): int {
+		$environment = $input->getArgument( 'environment' );
+		$factory = $this->getFactory( $environment );
+
+		$errorLogger = new ValidationErrorLogger();
+
+		$configLoader = new LoggingCampaignConfigurationLoader(
+			$factory->getCampaignConfigurationLoader(), $errorLogger
+		);
+		$factory->setCampaignConfigurationLoader( $configLoader );
+		$campaignCollection = $factory->getCampaignCollection();
+
+		if ( $campaignCollection ) {
```

Comment:
The CampaignCollection will be initialized with an empty array, but that is still a "truthy" value and the following code is run, resulting in the wrong error code if there were loading errors. Maybe addd `hasErrors` to `ErrorLogger` and check that instead?